script/test.sh

```diff
@@ -18,6 +18,12 @@ echo "Test local path as a git repository excluding the github check"
 ./vulcan-local -t . -e github -u file://./script/checktypes-stable.json
 echo "exit=$?"
 
+# Add a path and a tag to bypass check target validations.
+docker tag vulcan-local path/vulcan-local:xxx
+echo "Test local docker image"
+./vulcan-local -t path/vulcan-local:xxx -a DockerImage -i trivy -u file://./script/checktypes-stable.json
```

Comment:
For now this check fails (although it exits with code 0). Is this the intended behaviour? Are you planning to change it before merging?

Reply:
What is failing? path/vulcan-local:xxx ?
I don't scan vulcan-local because trivy has a check that I ask to remove [here](https://github.com/adevinta/vulcan-checks/commit/20a45052bb4adce923b1702665d6d04ea45dabf2)